### PR TITLE
DDO-1293 jenkins-helmfile-version-query: Update versions file path

### PR DESCRIPTION
**Pull request by choover-broad**
Update versions file path for https://github.com/broadinstitute/terra-helmfile/pull/1551

🥳

### CODE REVIEW

**Review by jack-r-warren — APPROVED** [review]

File: images/jenkins-helmfile-version-query/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM alpine:3.11.5
+FROM alpine:3
```
Comment (jyang-broad) [inline images/jenkins-helmfile-version-query/Dockerfile:1]: is this to fuzzy match everything after 3.xx.xx?

File: tools/Dockerfile [hunk 2]
```diff
@@ -2,9 +2,13 @@
 # Compile Go tools and install to /tools/bin
 #
 ARG GO_VERSION='1.16'
-ARG ALPINE_VERSION='3.13'
+ARG ALPINE_VERSION='3.14'
```
Comment (jyang-broad) [inline tools/Dockerfile:5]: pi!

**Review by jyang-broad — APPROVED** [review]
Look sane!

File: images/jenkins-helmfile-version-query/Dockerfile (hunk 1 above)
Comment (choover-broad) [reply images/jenkins-helmfile-version-query/Dockerfile:1]: yep

**Pull request merged**